packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -327,11 +327,11 @@ export function isManagedDOMElement(
 export function isValidEventTarget(
   target: EventTarget | ReactScopeMethods,
 ): boolean {
-  return typeof (target: any).addEventListener === 'function';
+  return typeof (target: Object).addEventListener === 'function';
 }
 
 export function isReactScope(target: EventTarget | ReactScopeMethods): boolean {
-  return typeof (target: any).getChildContextValues === 'function';
+  return typeof (target: Object).getChildContextValues === 'function';
```

Comment:
I'm curious. What benefit does this change provide?

Reply:
Looks nicer on the eyes. I've also had a lot of lemsip today, so it may have something to do with it too.